src/app/demo/typescript/typescript.component.ts

```diff
@@ -1,6 +1,6 @@
-import {Component} from '@angular/core';
-import {typeScriptWithConsoleLog, withDeps} from '../../exercise/helpers/helpers';
-import {ng2tsConfig} from '../../../../ng2ts/ng2ts';
+import { Component, HostBinding } from '@angular/core';
```

Comment:
I thought this is something the CLI is doing; is it not? I've been manually fixing it before the final PR up 'til now 😭